packages/circle-react-elements/src/components/WalletBalance/WalletBalance.stories.ts

```diff
@@ -3,7 +3,7 @@ import type { Meta, StoryObj } from '@storybook/react';
 import { WalletBalance } from './WalletBalance';
 
 const meta = {
-  title: 'WalletBalance',
+  title: 'Components/WalletBalance',
```

Comment:
we can namespace components to render in storybook under one category or create multiple different ones